### PR TITLE
Redirects for site migration

### LINKED ISSUE #277: [🚨 Announcement!] templates.netlify.app soon to be moved
Thank you for your interest in our template site. We have recently decided we are going to move Netlify templates to a dedicated integration page on [netlify.com](http://netlify.com/)! We will be redirecting users from netlify.templates to this page towards the end of September 2022 (specific details to come).

We still love the idea of working with partners throughout our ecosystem. If this sounds good to you, we have this document that will walk you through this partnership process: [Netlify Ecosystem Partner Template Process](https://asset.cloudinary.com/dzkoxrsdj/0058b841daff62a3bdb1886222692a70).

Please feel free to reach out to us at [templates@netlify.com](mailto:templates@netlify.com) if you have any questions or concerns.

Thank you!

### PR DESCRIPTION
**Pull request by hexagoncircle**
**- Summary**

Templates can now be discovered in the Integrations Hub on netlify.com at [netlify.com/integrations/templates](https://www.netlify.com/integrations/templates/). Notice had been given on [templates.netlify.com](https://templates.netlify.com/) that it will be deprecated and redirected to netlify.com. This PR introduces a `_redirects` file, which gets added to the output directory through a small eleventy config update that will pass the file through to the root.

* Related issue: https://github.com/netlify/marketing/issues/2312

**- Test plan**

* Review _redirect mapping_ tab in the [templates.netlify.com migration Google doc](https://docs.google.com/spreadsheets/d/1BHyulhfuZudDLjNcMaTbcQaxE4TNxWoYyncafDfOgAo/edit#gid=863206040)
* Ensure list of redirects are properly set up for domain-level redirects [[Netlify docs](https://docs.netlify.com/routing/redirects/redirect-options/#domain-level-redirects)]

**- Description for the changelog**

Adds page redirects to new home for templates on netlify.com

### CODE REVIEW

File: .eleventy.js [hunk 1]
```diff
@@ -2,41 +2,49 @@ const fetch = require("node-fetch");
...
-        .then(res => res.json())
-        .then(json => json.stargazers_count);
-
-      return template;
-    }));
+    );
   });
 
+  // Add redirects file to root of output directory
+  config.addPassthroughCopy("_redirects");
```
Comment (hexagoncircle) [inline .eleventy.js:38]: This is the main change to this file. The rest were done by prettier. I can revert if necessary!

**Comment by stephan1echung** [comment]
@MarcL @taty2010 It looks like we'd need to merge these changes to actually test the redirects, but please let us know if we can test redirects differently. Also, would either of you have the bandwidth to review this PR? If not, no worries, we'll go ahead and merge and test, and if it doesn't work as expected, we can revert and troubleshoot.

**Review by taty2010 — APPROVED** [review]
Looks good to me 👍🏾

Thanks for working on this!!

**Comment by MarcL** [comment]
@hexagoncircle @stephan1echung - Thanks for doing this.

re: testing - can we not use relative redirects rather than the full domain and then test on a deploy preview? I do this on my own site and I think it works.

**Comment by hexagoncircle** [comment]
> re: testing - can we not use relative redirects rather than the full domain and then test on a deploy preview?

It does work indeed – Thank you, @MarcL!

**Review by stephan1echung — APPROVED** [review]
Woohoo! I tested the deploy preview urls, and they all redirect to their correct url. :shipit:

File: README.md [hunk 2]
```diff
@@ -5,18 +5,14 @@ boilerplates.
 
 ## Important Notice
 
-:warning: Please be aware that [templates.netlify.com](https://templates.netlify.com/) pages will be redirected to relevant GitHub repos, [jamstack.org/](https://www.jamstack.org/), or [www.netlify.com](https://www.netlify.com/) pages. If you are interested in adding a template to [netlify.com/integrations/templates/](https://www.netlify.com/integrations/templates/), please reach out to templates@netlify.com
-
-
-
+:warning: [templates.netlify.com](https://templates.netlify.com/) is now being redirected to relevant GitHub repos, [jamstack.org/](https://www.jamstack.org/), or [www.netlify.com](https://www.netlify.com/) pages. If you are interested in adding a template to [netlify.com/integrations/templates/](https://www.netlify.com/integrations/templates/), please reach out to templates@netlify.com
```
Comment (stephan1echung) [inline README.md:8]: @hexagoncircle smol request to update the link for jamstack.org to `(https://jamstack.org/)`

**Pull request merged**